testing/helm/run.sh

```diff
@@ -57,7 +57,12 @@ fi
 if type kube-score >/dev/null 2> /dev/null; then
    echo Running kube-score...
    # Note we force all replicas to be > 1 as some checks are not done on replicas=1 cases e.g. antiaffinity
-   helm template $hpccchart ${options} | sed "s/replicas: 1/replicas: 2/" | kube-score score --output-format ci - >results.txt 2>errors.txt
+   helm template $hpccchart ${options} | sed "s/replicas: 1/replicas: 2/" | \
+     kube-score score --output-format ci \
+        --ignore-container-cpu-limit \
+        --ignore-container-memory-limit \
+        --ignore-test deployment-has-poddisruptionbudget \
```

Comment:
Should create a JIRA to come back and look at what it would take to re-enable this one.

Reply:
I have now done so